src/app/workbench/WorkbenchViews.tsx

```diff
@@ -22,127 +41,66 @@ export function WorkbenchViews({ index, type }: IWorkbenchViewsProps) {
   const { views, selection, commentsOpen, itemIDType } = ordino.workbenches[index];
   const [setRef] = useCommentPanel({ selection, itemIDType, commentsOpen, isFocused: type === EWorkbenchType.FOCUS });
 
-  let wb = null;
+  const [mosaicState, setMosaicState] = useState<any>(null);
```

Comment:
```suggestion
  const [mosaicState, setMosaicState] = useState<MosaicNode<string>>(null);
```
I think the correct type is `MosaicNode<string>`.